src/python/grpcio/grpc/compatibility.py

```diff
@@ -0,0 +1,6 @@
+import six
+
+if six.PY2:
```

Comment:
Our style guide does call for importing only modules and not classes, functions, or other elements of modules. And that will result in code that reads `from six import moves` and `moves.cStringIO`. That may not be the way most people write their code that uses the `six.moves` module, but there are not problems with it either, right?